Automattic/themes
# Try fixing the husky actions.

#### Changes proposed in this Pull Request:

Brought up by @matiasbenedetto here: p1644432879528019-slack-C029FM1EH, it looks like our pre-commit hook stopped working at some point. This PR updates it to use husky's v7 configuration (it looks like we were using v4).

**Instructions**
- `npm i`
- `composer update`

**To test**
- Make a change on this branch to some php file and commit it.
- Verify the linter runs and identifies and attempts to fix any errors.

#### Related issue(s):
